azure-pipeline-weekly.yml

```diff
@@ -8,33 +11,27 @@ schedules:
     - main
   always: true
 
-name: $(Date:yyyMMdd)$(Rev:rr)
-resources:
-  pipelines:
-  - pipeline: arcadeServicesInternalCI
-    source: arcade-services-internal-ci
-    branch: main
-
 stages:
   - stage: SynchronizeSecrets
+    displayName: Synchronize Secrets
     jobs:
     - job: Synchronize
-      pool: 
+      displayName: Synchronize Secrets
+      pool:
         name: NetCore1ESPool-Internal-NoMSI
-        demands: ImageOverride -equals 1es-windows-2019
+        demands: ImageOverride -equals 1es-windows-2022
       steps:
-      - task: UseDotNet@2
-        displayName: Install Correct .NET Version
-        inputs:
-          useGlobalJson: true
-
-      - script: dotnet tool restore
+      - powershell: |
```

Comment:
Is there something off with the `UseDotNet@2` task that we have to install `dotnet` manually?

Reply:
It's better to use the Arcade tooling for that because the .NET installation there is a bit smarter. For instance, if we wanted to use a tool built by the latest .NET 8 preview, it would probably fail to get it.